### DISCUSSION TITLE
GIF search/picker

### DISCUSSION THREAD

**Discussion by sverit** [stoatchat/discussions#640] [Q&A] [answered]
Only thing I am missing so far is a search and picker for GIFs :)

**Comment by llunariis** [comment, marked as answer]
there is a gif picker in the [new client](https://revolt.chat/app) via [gifbox](https://gifbox.me/)

**Reply by sverit** [reply]
Hi, thank you, but I dont't even see the Emoji button there, no GIF picker either?
![Screenshot 2023-12-02 171452](https://github.com/revoltchat/revolt/assets/31847769/078909b2-9ff9-4f84-b18a-5c3c6081e717)

**Reply by insertish** [reply]
You are about 2 days late to be able to enable it directly from the client (it will make a return over the coming few weeks), you can however just run:
```js
controllers.state.experiments.setEnabled('gif_picker', true)
```
in console to activate it manually.

**Reply by amycatgirl** [reply]
The limited demo release does not have those features. Like its name suggests, it's a limited demonstration of what the new client is going to look/feel, ~~not all features are present.~~

You can try those features out by hosting the client yourself.

Edit: i was wrong lmao
